docs/intro.md

```diff
@@ -1,34 +1,42 @@
 
-Our Wiki describes a general technology overview, the key advantages of Gear’s technology, helps to get started with Gear node setup and uploading your smart contracts, provides full API details and examples.
+Our Wiki outlines foundational information required, general technology overviews, the key advantages of Gear’s technology, help to get started with Gear node setup, how to upload your smart contracts, while also providing full API details and examples.
+
+All code, libraries, and tools are available on Github with a permissive Apache-2.0 license. Feel free to use the tools and libraries, log issues as you find them, or create pull requests for your bug-bears or features.
+
+As our project is developing and growing, contributions are more than welcome!
 
 ## What is Gear?
 
-Gear is a Substrate-based smart-contract platform allowing anyone to run dApp in a few minutes.
+Gear is a Substrate-based smart-contract platform that enables anyone to deploy a dApp in a matter of minutes. Gear is the most cost-effective way to run smart contracts that have been compiled from many popular programming languages, such as Rust, C, C++ and many more. It ensures very minimal, intuitive and sufficient API for running both newly written and existing programs on multiple networks without having to rewrite them. Smart contracts are stored in the blockchain’s state and are invoked, preserving their state upon request.
 
- - Gear provides Layer-1 decentralized network as the easiest and most cost-effective way to run WebAssembly programs (smart-contracts) compiled from many popular languages, such as Rust, C/C++ and more.
+Gear is planning on becoming a parachain in the Polkadot and Kusama networks to host smart contracts on these respective networks. This will mean that by deploying on Gear, developers would be able to take advantage of all the benefits of the Polkadot and Kusama networks and [ecosystems](/ecosystem/polkadot.md) at minimal cost.
 
- - Gear ensures very minimal, intuitive, and sufficient API for running both newly written and existing programs on multiple networks without the need to rewrite them. Smart Contracts are stored in the blockchain’s state and are invoked preserving their state upon request.
+Gear will assist with the transition to mass use of Web3 technologies by enabling the running of innovative dApps, microservices, middleware and open APIs.
 
-  - Gear is intended to be a Layer-2 parachain for hosting smart contracts on the Polkadot/Kusama network. This allows anyone to deploy any dApp on Polkadot and Kusama to take advantage of all the benefits of their ecosystems, yet at the minimal financial expense. Also, refer to the [Ecosystem](/ecosystem/polkadot.md) chapter for more details about Polkadot and layers definition.
+## How to engage with the Gear community
```

Comment:
Could you add our telegram group here?

Reply:
Sure, added 
https://github.com/gear-tech/wiki/pull/46/commits/271650c2a3f5e0cfd4e7dc956060761c172b31f4